tests/unit/extensions/elasticsearch/ActiveRecordTest.php

```diff
@@ -443,24 +429,22 @@ public function testBooleanAttribute()
         $this->assertEquals(2, count($customers));
     }
 
-    public function testScriptFields()
-    {
+    public function testScriptFields() {
         $orderItems = OrderItem::find()->fields([
-            'quantity',
-            'subtotal',
-            'total' => [
-                'script' => "doc['quantity'].value * doc['subtotal'].value",
-                'lang' => 'groovy',
-            ]
-        ])->all();
+                    'quantity',
```

Comment:
Please keep the original indentation (similar in the following code). In general, try not to fix the coding style of the original code while you are fixing a bug. You may submit a separate PR is coding style needs to be adjusted massively. Thanks.